lib/core/environment.dart

```diff
@@ -0,0 +1,14 @@
+class Environment {
+  static const Environment staging = Environment();
```

Comment:
just curious when will you apply flavor to platforms? In other PR? 🙏

Reply:
On it 👍

Reply:
I will do it in other PR due to building failure in iOS, cause it will take more time to investigate 😢

Reply:
@manh-t I added it in this PR 😁